e2e/cypress/integration/system_console/search_teams_spec.js

```diff
@@ -38,29 +38,31 @@ describe('Search teams', () => {
         cy.get('[data-test=search-input]').type(displayName + '{enter}');
 
         // page 1 has PAGE_SIZE items
-        cy.get('[data-test=team-display-name]').as('displayNames').should('have.length', PAGE_SIZE);
+        cy.get('[data-test=team-display-name]').should('have.length', PAGE_SIZE);
```

Comment:
Instead of using a data-test attribute, we add an attribute data-testid to the element and then in cypress, we use it something like this:
```cy.findByTestId('team-display-name')```

Can you please update this throughout.

Reply:
Updated except for two spots that are using aliases.